### PR TITLE
Add Tikui icon

### PR DESCRIPTION
**Pull request by hugojeanmaire1**
Thanks to @Gnuk and @nonomoho for helping

### CODE REVIEW

File: assets/icons/files/tikui.svg
Comment (mallowigi) [inline assets/icons/files/tikui.svg:21]: please remove this

File: assets/icons/files/tikui.svg
Comment (mallowigi) [inline assets/icons/files/tikui.svg:11]: Icons should have only one color, so please keep only the #310ca3, put it in the topmost `g` and add the `data-iconColor="Tikui"` property so that it's color can be changed

File: assets/icons/files/tikui.svg
Comment (mallowigi) [inline assets/icons/files/tikui.svg:2]:
```suggestion
<svg width="16px" height="16px" data-big="true" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

File: assets/icons/files/tikui.svg
Comment (mallowigi) [inline assets/icons/files/tikui.svg:1]: remove this

File: assets/icons/files/tikui.svg
Comment (mallowigi) [inline assets/icons/files/tikui.svg:3]: remove this

File: icon_associations.xml
Comment (mallowigi) [inline icon_associations.xml:51]: Put it in the "T" section, alphabetically in order with the other files. And don't use a priority of 1000.

File: icon_associations.xml
Comment (mallowigi) [inline icon_associations.xml:49]:
```suggestion
           pattern="^tikuiconfig\.json$"
```

File: assets/icons/files/tikui.svg
Comment (mallowigi) [inline assets/icons/files/tikui.svg:10]: please remove ids

**Review by mallowigi — CHANGES_REQUESTED** [review]

**Review by mallowigi — APPROVED** [review]

**Comment by hugojeanmaire1** [comment]
Do we need to do something more to add the icon @mallowigi ?

**Comment by mallowigi** [comment]
You can merge, but FYI this will not be deployed until I have some time to deploy a new build.

**Comment by hugojeanmaire1** [comment]
We can't merge it. We don't have write access to the repository 😒

**Pull request merged**
